src/SFML/Graphics/Rect.cpp

```diff
@@ -30,6 +30,21 @@
 #include <SFML/Internal.h>
 
 
+namespace
+{
+sfVector2f toCType(const sf::Vector2f& vector)
+{
+    sfVector2f vec = {vector.x, vector.y};
+    return vec;
```

Comment:
Why use temporary variables here? Why not:
```c++
return {vector.x, vector.y};
```

Reply:
That requires C++11. Because SFML 2 uses C++03 I'm assuming that CSFML 2 inherits that same restriction.

Reply:
Ah I see. Then i'll approve of this.